tests/sles4sap/hana_install.pm

```diff
@@ -24,7 +25,7 @@ sub upload_install_log {
 
 sub run {
     my ($self) = @_;
-    my ($proto, $path) = $self->fix_path(get_required_var('MEDIA'));
+    my ($proto, $path) = $self->fix_path(get_required_var('HANA'));
```

Comment:
I don't which is better: to use `NW` for NetWeaver media and `HANA` media or `MEDIA` variable for both cases?

Nothing important but this is something I have in mind since some time now :-)

Reply:
> Looks promising!
> 
> If I correctly understand we will be able to either install:
> 
> * a standalone NW
> * a standalone HANA
> * a NW cluster

We have these already.

> * a HANA cluster

Yes.

> * a NW cluster with HANA database (in cluster too)

No.  At least not yet...  The scope of this PR is only testing HANA SR.

Reply:
PD:

In another PR, @alvarocarvajald prefered having 2 variables to distinguish HANA from NW instead of MEDIA, and I think he's right.  Unless we create another variable to differentiate them, both of these have overloaded semantics, as it is used also to get the URI of the media.

Reply:
>We have these already.

Yes I know, it was just to summarize all :-)

>No. At least not yet... The scope of this PR is only testing HANA SR.

So in that case you are reusing the `netweaver_cluster.pm` file to create a HANA cluster, right? In that case it may be better to rename this file to something like `sap_cluster.pm` to avoid any confusion, WDYT? (and same for `netweaver_network.pm` or other NW files that are not NW specific anymore).

Reply:
>In another PR, @alvarocarvajald prefered having 2 variables to distinguish HANA from NW instead of MEDIA, and I think he's right. Unless we create another variable to differentiate them, both of these have overloaded semantics, as it is used also to get the URI of the media.

Ok, and if we want to add HANA in NW cluster we will need the two paths in fact.

Reply:
> > We have these already.
> 
> Yes I know, it was just to summarize all :-)
> 
> > No. At least not yet... The scope of this PR is only testing HANA SR.
> 
> So in that case you are reusing the `netweaver_cluster.pm` file to create a HANA cluster, right? In that case it may be better to rename this file to something like `sap_cluster.pm` to avoid any confusion, WDYT? (and same for `netweaver_network.pm` or other NW files that are not NW specific anymore).

Yes. The renaming will be done as the last step when everything's working ok.

Reply:
> PD:
> 
> In another PR, @alvarocarvajald prefered having 2 variables to distinguish HANA from NW instead of MEDIA, and I think he's right. Unless we create another variable to differentiate them, both of these have overloaded semantics, as it is used also to get the URI of the media.

Exactly. With `HANA` and `NW` we can do boolean check (i.e.: `get_var('HANA')`) as well as get the address to the media (i.e.: `my $share = get_var('HANA')`) with the same variable. OTOH, if we use `MEDIA` for both, we will still need at least another variable to know which product we're installing, as they need to be treated differently. So we either go with:

```
my $share = get_var('MEDIA');
if (check_var('SAP_PRODUCT', 'hana') {
    # do HANA stuff
}
elsif (check_var('SAP_PRODUCT', 'nw') {
    # de NW stuff
}
```

Or with:

```
my $share;
if ($share = get_var('HANA')) {
    # do HANA stuff
}
elsif ($share = get_var('NW')) {
    # do NW stuff
}
```

I prefer the second option as only one variable (either NW or HANA) is required to be defined in the test suites.